pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1054,6 +1054,12 @@ func (og *operationGenerator) GenerateMapVolumeFunc(
 			klog.Infof(volumeToMount.GenerateMsgDetailed("MapVolume.WaitForAttach succeeded", fmt.Sprintf("DevicePath %q", devicePath)))
 
 		}
+		// For CSI driver, WaitForAttach returns attachment name as devicePath,
+		// not actual devicePath.
```

Comment:
can you also add a little note here to say that for CSI the devicePath is returned by the below SetUpDevice instead.

Reply:
NVM, see comment below about the "// A plugin doesn't have attacher" comment. If we can clarify that comment then we don't need a duplicate comment above.